### ISSUE TITLE
Cannot change audio track on Android TV

### ISSUE THREAD

**Issue by hariseldon99** [jellyfin/jellyfin#3905]
**Server System (please complete the following information):**
 - OS: Ubuntu 18.04. No docker
 - Jellyfin Version: 10.6.2
 - Hardware settings & device: VAAPI on Intel i3-2310M
 - Reverse proxy: None
 - Other hardware notes: None
 
**Client System (please complete the following information):**
 - Device: Sony Bravia KD-43X7500F
 - OS:  Android
 - Client: Android
 - Client and Browser Version: 0.11.4

Basically exactly the same as this problem:
https://www.reddit.com/r/jellyfin/comments/ggzpsi/cannot_change_audio_track_on_android_tv/

Happens with any media that has multiple audio tracks. Trying to change the track causes Jellyfin client to crash and reload

jellyfin logs: http://paste.ubuntu.com/p/gGXRrfTHdV/
ffmpeg logs: http://paste.ubuntu.com/p/WrHGJFHxWN/
